Assignment-1/code/style.css

```diff
@@ -1 +1,121 @@
+.subdiv
+{
+   width:99vw;
+   height:33vh;
 
+}
+@media screen and (min-width:1025px)
+{
+    .first-div
```

Comment:
In This session (i.e having two divs )

Here also the padding was not good it is not getting 100% width there is space on the left side of the pages ( pitular on 768 view )


please make this changes